example-files/mango-google-cloud.ipynb

```diff
@@ -0,0 +1,115 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bdgenomics.adam.adamContext import ADAMContext\n",
+    "from bdgenomics.mango.rdd import GenomicVizRDD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ac = ADAMContext(sc)\n",
+    "genomicRDD = GenomicVizRDD(sc)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "alignmentFile = \"gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/data/150140/alignment/150140.chrom20.ILLUMINA.bwa.CHM1.20131218.bam\""
```

Comment:
what other datasets are hosted besides 1000g? I ask because 1000g is the only datasource on AWS, so it would be nice to reference a different one in the GCE examples